tools/psoxy-test/lib/aws.js

```diff
@@ -246,6 +247,10 @@ async function upload(bucket, key, file, options, client) {
     commandOptions.ContentEncoding = 'gzip';
   }
 
+  if(path.extname(key)?.toLowerCase() === '.csv') {
+    commandOptions.ContentType = 'text/csv';
```

Comment:
OK, not sure required for anything but good practice. I think can ALSO add a `.ndjson` case